client/web-sveltekit/src/lib/search/resultsIndicator/InfoBadge.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+    import { getProgressText } from '$lib/branded'
+    import type { Progress } from '$lib/shared'
+
+    export let searchProgress: Progress
+    export let state: 'error' | 'complete' | 'loading'
+
+    $: isError = state === 'error'
+</script>
+
+<div class={`progress-message ${isError && 'error-text'}`}>
```

Comment:
Take a look a [this](https://learn.svelte.dev/tutorial/classes)

```suggestion
<div class="progress-message" class:error-text={isError}>
```

Reply:
Looks like there are a couple of other places where we can avoid string interpolation as well